v2/googlecloud-to-googlecloud/docs/BigtableChangeStreamsToGcs/README.md

```diff
@@ -0,0 +1,16 @@
+# Bigtable Change Streams To Google Cloud Storage Dataflow Template
+
+The [BigtableChangeStreamsToGCS]
+(src/main/java/com/google/cloud/teleport/v2/templates/BigtableChangeStreamsToGCS.java)
+pipeline reads messages from Cloud Bigtable Change Streams and stores them in a
+Google Cloud Storage bucket using the specified file format.
+
+Data will be bucketed by processing timestamp into different windows. By default, the
+window size is 5 minutes.
+
+The data can be stored in a Text or Avro File Format.
+
```

Comment:
Shall we follow the structure / content of https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/googlecloud-to-googlecloud/docs/SpannerChangeStreamsToGcs/README.md  and make ours similar?

Looks like we're picking the random factoids to put into the README file here.

Reply:
I added the basics here, which are the same for Bigtable and Spanner. 
I'll be finalizing before I submit my PR to the main fork.